BoxContentSDK/BoxContentSDK/Requests/BOXFolderPaginatedItemsRequest.m

```diff
@@ -90,12 +116,21 @@ - (void)performRequestWithCompletion:(BOXItemArrayCompletionBlock)completionBloc
                                                                                       ancestors:pathFolders];
             }
             [BOXDispatchHelper callCompletionBlock:^{
-                completionBlock(items, totalCount, NSMakeRange(offset, limit), nil);
+                refreshBlock(items, totalCount, NSMakeRange(offset, limit), nil);
+                
+                // Don't want to update cache for a subclass recursively calling this method.
```

Comment:
I agree that it would be good to update the cache here, but I haven't figured out a good way to differentiate it, since it still gets passed as the subclass to the cache update method. Would be good to add in the future.